odoo/sql_db.py

```diff
@@ -278,49 +237,48 @@ class Cursor(BaseCursor):
     """
     IN_MAX = 1000   # decent limit on size of IN queries - guideline = Oracle limit
 
-    def __init__(self, pool, dbname, dsn, serialized=True):
+    def __init__(self, pool, dbname, dsn, **kwargs):
         super().__init__()
-
+        if 'serialized' in kwargs:
+            warnings.warn("Since 16.0, 'serialized' parameter is not used anymore.", DeprecationWarning, 2)
+        assert not (kwargs.keys() - {'serialized'})
```

Comment:
```suggestion
        assert kwargs.keys() <= {'serialized'}
```
might be clearer and doesn't need to create a third set. `assert` doesn't use the value for an error message anyway (in the standard python) so this makes not difference to the final error message.